vm/src/frame.rs

```diff
@@ -80,6 +81,7 @@ enum UnwindReason {
 }
 
 #[pyclass]
+#[derive(Clone)]
```

Comment:
This might be tricky, since the traceback will contain a copied frame, and not the actual frame. The solution might be to modify the run method to take a clonable `PyFrameRef`.

Reply:
Sure. I will merge this and we can do that in the future